### PR TITLE
6 delegations types

### PR DESCRIPTION
**Pull request by OsamaNajjar**
Dear @SameerAlHarbi 
Delegations Types done
Plaese check and aprove pull requast

### CODE REVIEW

**Review by SameerAlHarbi — CHANGES_REQUESTED** [review]

File: src/routers/delegations.router.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const express = require('express');
+const delegationsController = require('../controllers/delegations.controller');
 
 const Router = express.Router();
 
+// /delegations/delegationsTypes?{lang=A} => GET
+Router.get('/delegationsTypes', delegationsController.getDelegationsTypes);
```
[the remark this answers is not in the document]
Comment (OsamaNajjar) [reply src/routers/delegations.router.js:7]: done

File: src/managers/delegations.manager.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const sapPool = require('../util/sap-rfc');
+const modelMapper = require('../models/model-mapper');
+const delegationType = require('../models/delegationType.model');
+
+/**
+ * Get an array of all delegationsTypes in SAP system.
+ * @param {string} lang - Results language.
+ * @return {Array<delegationType>} Array of delegationsTypes objects.
+ */
+ exports.getDelegationsTypes = async (lang = 'A') => {
```
[the remark this answers is not in the document]
Comment (OsamaNajjar) [reply src/managers/delegations.manager.js:10]: done

File: src/controllers/delegations.controller.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const delegationsManager = require('../managers/delegations.manager');
+
+exports.getDelegationsTypes = async (req, res, next) => {
```
[the remark this answers is not in the document]
Comment (OsamaNajjar) [reply src/controllers/delegations.controller.js:3]: done

File: src/controllers/delegations.controller.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    
+    const lang = req.query.lang;
+
+    try {
+
+        const results = await delegationsManager
+            .getDelegationsTypes(lang);
```
[the remark this answers is not in the document]
Comment (OsamaNajjar) [reply src/controllers/delegations.controller.js:10]: done

**Comment by SameerAlHarbi** [comment]
merge to development

**Pull request closed without merging**